packages/react-core/src/filters/tileFeaturesGeometries.js

```diff
@@ -4,36 +4,87 @@ import booleanWithin from '@turf/boolean-within';
 import intersect from '@turf/intersect';
 import transformToTileCoords from '../utils/transformToTileCoords';
 import { TILE_FORMATS } from '../types';
+import transformTileCoordsToWGS84 from '../utils/transformTileCoordsToWGS84';
+
+export const GEOM_STORED_VALUE = '__geomValue';
```

Comment:
I'm a bit concerned about this `GEOM_STORED_VALUE` because it's a bit hard to determine what it represents 🤔 

Given that, I'll suggest we replace this with something more meaningful, like`TILESET_GEOMETRY_PROPERTY`, which makes it more obvious that it's a Geometry Property from a Tileset Data Source, WDYT?

Reply:
TILESET sounds like static tileset and this code could be used for dynamic tiling. I've changed to `FEATURE_GEOM_PROPERTY `